### PR TITLE
chore: add preset for federation agent repositories

### PR DESCRIPTION
**Pull request by jgiovaresco**

### CODE REVIEW

File: federationagent.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "extends": ["config:base", ":label(dependencies)"],
+    "rebaseWhen": "conflicted",
+    "prConcurrentLimit": 5,
+    "commitMessageTopic": "{{depName}}",
```
Comment (phiz71) [inline federationagent.json:5]: Should we add the same fields on lib & policy config files?

**Review by phiz71 — APPROVED** [review]
LGTM.

File: federationagent.json (hunk 1 above)
Comment (jgiovaresco) [reply federationagent.json:5]: we can 🤷  but that's not the purpose of this PR :p

Comment (phiz71) [reply federationagent.json:5]: yes I know, it was just a thought I had

**Pull request merged**